sed/binning/numba_bin.py

```diff
@@ -55,7 +55,11 @@ def _hist_from_bin_range(
         is_inside = True
         flatidx = 0
         for i in range(ndims):
-            j = (sample[t, i] - ranges[i, 0]) * delta[i]
+            # strip off numerical rounding errors
+            j = round((sample[t, i] - ranges[i, 0]) * delta[i], 11)
```

Comment:
I'm not sure I understand what you mean. We do create the array in advance (line 45). The round here happens to strip off numerical round errors that happen for certain combinations of edge values. 
The underlying reason is that here we do the forward transformation (position -> bin index), while in np.histogramdd linspace is used to create the bin edges, i.e. (bin index -> position). These two directions can produce slightly different results due to numerical float rounding errors, which I strip off at the 11th digit with this round command, which emperically does the trick, at least for the test case I identified.

Reply:
> I'm not sure I understand what you mean. We do create the array in advance (line 45). The round here happens to strip off numerical round errors that happen for certain combinations of edge values. The underlying reason is that here we do the forward transformation (position -> bin index), while in np.histogramdd linspace is used to create the bin edges, i.e. (bin index -> position). These two directions can produce slightly different results due to numerical float rounding errors, which I strip off at the 11th digit with this round command, which emperically does the trick, at least for the test case I identified.

Sorry, my bad. I read that `j` was an array instead of a single float. As I said, as `round` is now supported in numba, the change makes sense.